source/docs/tuner/tuner-swerve/validating-drivetrain.rst

```diff
@@ -0,0 +1,73 @@
+Validating the Drivetrain
+=========================
+
+Tuner performs several validation procedures to confirm inverts, offsets and mechanical functionality of the drivetrain.
+
+This test consists of two steps:
+
+* Verify Drive
+* Verify Steer
+
+.. image:: images/swerve-verification-page.png
+   :alt: Picture of the verification page in Tuner
+
+Verify Drive
```

Comment:
We should make it verify steer before the drive first in Tuner, and the documentation should be steer first, drive second to match.